compiler/damlc/tests/daml-test-files/Iou_template.EXPECTED.md

```diff
@@ -1,58 +1,58 @@
-# Module Iou_template
-
-
+# <a name="module-ioutemplate-98694"></a>Module Iou\_template
 
 ## Templates
 
-### Template `Iou`
-
-
-| Field        | Type/Description |
-| :----------- | :----------------
-| `issuer`     | `Party` |
-| `owner`      | `Party` |
-| `currency`   | `Text` |
-|              | only 3-letter symbols are allowed |
-| `amount`     | `Decimal` |
-|              | must be positive |
-| `regulators` | `[` `Party` `]` |
-|              | `regulators` may observe any use of the `Iou` |
+### <a name="template-ioutemplate-iou-32396"></a>Template Iou
 
+| Field      | Type/Description |
+| :--------- | :----------------
+| issuer     | Party |
+| owner      | Party |
+| currency   | Text |
+|            | only 3-letter symbols are allowed |
+| amount     | Decimal |
+|            | must be positive |
+| regulators | \[Party\] |
+|            | `regulators` may observe any use of the `Iou` |
 
 
-  #### Choices
+Choices:
 
-* `Merge`
+* Merge
+  
   merges two "compatible" `Iou`s
-
-  | Field      | Type/Description |
-  | :--------- | :----------------
-  | `otherCid` | `ContractId` `Iou` |
-  |            | Must have same owner, issuer, and currency. The regulators may differ, and are taken from the original `Iou`. |
-
-* `Split`
+  
+  | Field    | Type/Description |
+  | :------- | :----------------
+  | otherCid | ContractId Iou |
```

Comment:
Why is this no longer formatted as code? The previous behavior seemed sensible to me.

Reply:
I guess it might be because it can now include links? In that case I think we should still have it in some monospace font.

Reply:
Well, to add links in types (markdown doesn't let you mix code formatting with links), and to be a little closer to what the Rst renderer does. I think I'll defer worrying about font formatting to if/when we are trying to replace the Rst website.